pyrene/src/components/TabView/TabView.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { Story, Meta } from '@storybook/react';
+import Icon from '../Icon/Icon';
+import TabViewComponent, { TabViewProps } from './TabView';
+import StateProvider from '../../examples/StateProvider';
+import Placeholder from '../../examples/Placeholder';
+
+export default {
+  title: 'Components/Layout/TabView',
+  component: TabViewComponent,
+} as Meta;
+
+const renderAuxiliaryIcon = (name: string) => (
+  <div style={{ marginLeft: 4 }}>
+    <Icon name={name} />
+  </div>
+);
+
+type ProviderState = {
+  tabName?: number | undefined;
+};
+
+const Template: Story<TabViewProps> = () => (
+  <StateProvider<TabViewProps, ProviderState> initState={{ tabName: undefined }}>
+    {
+      (state, setState) => ({
```

Comment:
Did you consider changing the API of `StateProvider` to directly return a react component here instead of an object? I think that would be more react-like and also allow to return nested components (maybe this is needed for another story?) or basically anything that is valid JSX.

For example:
```jsx
(state, setState) => (
  <TabViewComponent
    initialTabName="Tab 1"
    directAccessTabs={3}
    ...
  />
)
```